### DISCUSSION TITLE
Separate selectable categories for Consoles / Systems and Arcade / Single Game cores

### DISCUSSION THREAD

**Discussion by detheavn** [RetroDriven/Pocket_Updater#28] [Ideas]
As someone who would prefer to wait using all the arcade cores until Analogue implements a way to group these under 1 category, I'd love to see an category split. I created a little mockup below as an example :)

![image](https://user-images.githubusercontent.com/11292396/194907953-37ca610c-977d-4b9c-8514-9723aaf6f503.png)

**Comment by Compeau** [comment]
The info to do this is already in the source json, each core has a property named "category" that can be "handheld", "console", "arcade", or "computer".  Filter console and handheld into one list, arcade and computer into another.

**Reply by Compeau** [reply]
I made this change, but apparently I don't have permissions to push my branch to the repo.
